<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTransformWidget</class>
 <widget class="QWidget" name="ViewTransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>32</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="btn_view_fit">
     <property name="toolTip">
      <string>Fit View</string>
     </property>
     <property name="text">
      <string>Fit View</string>
     </property>
     <property name="icon">
      <iconset theme="zoom-fit-best">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Zoom</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spin_zoom">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btn_zoom_in">
     <property name="toolTip">
      <string>Zoom In</string>
     </property>
     <property name="text">
      <string>Zoom In</string>
     </property>
     <property name="icon">
      <iconset theme="zoom-in">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btn_zoom_out">
     <property name="toolTip">
      <string>Zoom Out</string>
     </property>
     <property name="text">
      <string>Zoom Out</string>
     </property>
     <property name="icon">
      <iconset theme="zoom-out">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btn_zoom_reset">
     <property name="toolTip">
      <string>Reset Zoom</string>
     </property>
     <property name="text">
      <string>Reset Zoom</string>
     </property>
     <property name="icon">
      <iconset theme="zoom-original">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Angle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spin_angle">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>359.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btn_angle_reset">
     <property name="toolTip">
      <string>Reset Rotation</string>
     </property>
     <property name="text">
      <string>Reset Rotation</string>
     </property>
     <property name="icon">
      <iconset theme="rotation-allowed">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btn_flip_view">
     <property name="toolTip">
      <string>Flip View</string>
     </property>
     <property name="text">
      <string>Flip View</string>
     </property>
     <property name="icon">
      <iconset theme="object-flip-horizontal"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
